packages/ua-devtools/src/oapp/config.ts

```diff
@@ -75,10 +73,41 @@ export const configureOAppPeers: OAppConfigurator = async (graph, createSdk) =>
 
                 logger.verbose(`Creating a connection ${formatOmniVector({ from, to })}`)
                 return [await sdk.setPeer(to.eid, to.address)]
-            })
+            },
+            {
+                onStart: (
+                    logger,
+                    [
+                        {
+                            vector: { from, to },
+                        },
+                    ]
+                ) => logger.verbose(`Checking connection for ${formatOmniVector({ from, to })}`),
```

Comment:
Where would these actually be printed? Is it in the terminal when wiring 2 contracts together using `npx hardhat lz:oapp:wire --oapp-config layerzero.config.ts` ? If so, I don't see any of these logs (even before I made my changes) -- perhaps there is a flag for a more verbose logging level similar to `forge`'s `-vvvv`?

Reply:
The log levels for our hardhat tasks are specified using `--log-level` CLI flag so in your case:

```bash
npx hardhat lz:oapp:wire --oapp-config layerzero.config.ts --log-level verbose
```

If you do

```bash
npx hardhat lz:oapp:wire --help
```

it should give you all the possible values of `-log-level`

Reply:
Oo this is exactly what I was looking for, thank you!